src/panel-triggers/components/AckModal.tsx

```diff
@@ -66,6 +66,16 @@ export class AckModalUnthemed extends PureComponent<Props, State> {
       loading: false,
     };
   }
+  
+  getSeverityOptions(texts: any) {
```

Comment:
Please add here a comment that 0 Not classified is omitted on purpose

Reply:
Added the comment.